swagger.yaml

```diff
@@ -1113,12 +1113,43 @@ definitions:
         $ref: '#/definitions/OpenShiftResources'
       openshift_host_env:
         type: string
-      userId:
-        type: string
       image_registry:
         type: string
+      persistent_volumes:
+        type: array
+        items:
+          $ref: '#/definitions/PersistentVolume'
     required:
     - id
+  PersistentVolume:
+    type: object
+    properties:
+      id:
+        type: integer
+        format: int64
+      name:
+        type: string
+      capacity:
+        type: integer
+        format: int64
+      access_modes:
+        type: array
+        items:
+          type: string
+          enum:
+          - ReadWriteOnce
+          - ReadOnlyMany
+          - ReadWriteMany
+      nfs_path:
+        type: string
+      nfs_server:
+        type: string
+      reclaim_policy:
```

Comment:
should be an enum with `Retain` and `Recycle`
https://docs.openshift.com/container-platform/3.3/architecture/additional_concepts/storage.html#pv-recycling-policy